packagelist

```diff
@@ -101,7 +101,7 @@ enpass-deb
 epson-inkjet-printer-stylus-nx110-series-deb
 epson-printer-utility-deb
 exa-git
-exercism
+exercism-cli-bin
```

Comment:
That was sloppy of me, thanks for fixing!

Reply:
It's quite likely I'll keep forgetting packagelist so I added a commit hook that automates this in https://github.com/pacstall/pacstall-programs/pull/5139

Reply:
Thanks!